src/components/OpenSource/Contributions.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import styled from 'styled-components'
+import BlueBackground from '../Common/BlueBackground'
+import ContributionsCopy from './ContributionsCopy'
+import { Grid, Row, Col } from '../grid'
+import { Repo } from '../Repo'
+import { generate } from 'shortid'
+import StyledLink from '../Common/StyledLink'
+import Image from '../Common/Image'
+import { BodyPrimary } from '../Typography'
+
+const Wrapper = styled(BlueBackground)`
+  padding: ${props => props.theme.spacing[4]} 0;
+`
+
+const StyledBodyPrimary = styled(BodyPrimary)`
+  padding: ${props => props.theme.spacing[3.5]} 0;
+`
+
+const RepoWrapper = styled.div`
+  padding-bottom: ${props => props.theme.spacing[3]};
+`
+
+const StyledImage = styled(Image)`
+  max-width: 54px;
+`
+
+const Contributions = ({
+  contentfulOpenSourcePage: {
+    contributionsSectionImage,
+    contributionsSectionDescription,
+    contributionsSectionCtaText,
+    contributionsSectionCtaLink,
+    contributionsSectionGithubRepos: repos,
+    contributionsSectionTitleLine1,
+    contributionsSectionTitleLine2,
+    contributionsSectionTitleLine3,
+    openSourceMetaRepoCount,
+    openSourceMetaPullRequestCount
+  }
+}) => {
+  const contributions = {
+    contributionsSectionTitleLine1,
+    contributionsSectionTitleLine2,
+    contributionsSectionTitleLine3,
+    openSourceMetaRepoCount,
+    openSourceMetaPullRequestCount
+  }
+
+  return (
+    <Wrapper>
+      <Grid>
+        <Row>
+          <Col width={[1, 1, 1, 1, 6 / 12]}>
```

Comment:
Column width is not quite as simple as `6/12` for everything other than mobile. (Same for contributionsSectionDescription below)

For example, large tablet:
![image](https://user-images.githubusercontent.com/15656538/58712131-e40fb800-83b7-11e9-8c6c-9517a06d26aa.png)